docs/dind.md

```diff
@@ -0,0 +1,49 @@
+# rke DinD
+
+Docker-in-Docker (DinD) feature is created mainly for testing.
+
+DinD creates docker containers locally which are then used as cluster nodes.
+
+It still requires a cluster.yml with minimal information:
+
+```
+kubernetes_version: "$KUBE_VERSION"
+nodes:
+- address: foo
+  role:
+  - controlplane
+  - worker
+  - etcd
+  user: bar
+```
+where KUBE_VERSION is any version in the output list of the command `rke config --list-version --all`. Note that in multinode clusters, the value of address must be unique
+
+To create a cluster with DinD execute:
+`rke up --dind`
+
+After the creation is completed, you will see one docker container per node running. You can use the created `kube_config_cluster.yml` to access the cluster as in a normal RKE deployment
+
+To remove a cluster with DinD execute:
+`rke remove --dind`
+
+The docker containers should be removed completely
+
+# FAQ
+
+* Do I need to create a network for the docker containers?
+
+No. The created docker containers use the default docker bridge network which allows to connect from the host to the containers on IPs in the subnet "172.17.0.0/16". However, they have no ssh service running, therefore you `docker exec` if you want to connect to them
+
+* What image is used for those containers?
+
+The default docker image mirrores in rancher registry: rancher/docker:${VERSION}-dind that is used for that
```

Comment:
Please share where you got this from, the image is not mirrored to rancher or coming from Rancher. It is upstream's Docker image.